draft-ietf-quic-transport.md

```diff
@@ -2432,17 +2432,14 @@ The capacity available on the new path might not be the same as the old path.
 Packets sent on the old path MUST NOT contribute to congestion control or RTT
 estimation for the new path.
 
-On confirming a peer's ownership of its new address, an endpoint MUST
+On confirming a peer's ownership of its new address, an endpoint SHOULD
 immediately reset the congestion controller and round-trip time estimator for
 the new path to initial values (see Appendices A.3 and B.3 in {{QUIC-RECOVERY}})
-unless it has knowledge that a previous send rate or round-trip time estimate is
-valid for the new path.  For instance, an endpoint might infer that a change in
-only the client's port number is indicative of a NAT rebinding, meaning that the
-new path is likely to have similar bandwidth and round-trip time. However, this
-determination will be imperfect.  If the determination is incorrect, the
-congestion controller and the RTT estimator are expected to adapt to the new
-path.  Generally, implementations are advised to be cautious when using previous
-values on a new path.
```

Comment:
I think we should retain some of this to deal with the bit where people decide to ignore the "SHOULD".

Reply:
Thanks. Tried to address this in the current revision to the text.